README.md

```diff
@@ -7,21 +7,43 @@ data from `localhost:8000` **where the Django app must be running**.
 
 ## Django
 
-**Install Prerequisites**
-- `pip install -r requirements.txt`
+### Install Dependencies
+1. Install `pipenv`
 
-**Create dotenv file with app secret**
+    - **Homebrew**:
+
+        - `brew install pipenv`
+
+    - **pip**:
+
+        - `pip install pipenv`
+        - or depending on your versioning setup:
+        - `pip3 install pipenv`
+
+        - You can install at the User level using **pip** via: `pip install --user pipenv`
+
+2. `cd` to top level of project (contains `Pipfile` and `Pipefile.lock`)
+
+3. Install dependencies
+
+    - `pipenv install`
+
+### Installing new packages
+- Simply install via: `pipenv install [package-name]`
+
+### Create dotenv file with app secret
 - `echo "SECRET_KEY='TEMPORARY SECRET KEY'" > vp/.environment`
 
-**Start dev server from app root**
+### Start dev server from app root
 - `cd vp`
```

Comment:
Missing `pipenv shell` step. Need to execute before `cd vp`, otherwise complains about missing modules.

Reply:
`pipenv shell` should only be needed to spawn the virtualenv shell. Wouldn't be needed to install the requirements at the `visual-programming` level. What are the warnings you're getting? 

You can change directories around and still start up a Pipenv shell, too.

Reply:
Sorry, I worded that poorly. Starting the server without the `pipenv shell` or `pipenv run` steps you added to the README produced a `ModuleNotFoundError`; I had only tried the `shell` option. All is good now 👍